cpp/cmake/Dependencies.cmake

```diff
@@ -199,7 +199,7 @@ if(BUILD_GTEST)
 	include(ExternalProject)
 	ExternalProject_Add(googletest
 	  GIT_REPOSITORY    https://github.com/google/googletest.git
-	  GIT_TAG           6ce9b98f541b8bcd84c5c5b3483f29a933c4aefb
+	  GIT_TAG           release-1.10.0
```

Comment:
Any reason not to use the commit SHA rather than the tag? I ask because I've been burned in the past by other projects moving their tags. Don't suspect it would be an issue with gtest, but it seems safer.

Reply:
I 100% agree with you, I tend to hate tags. But in this case the nice thing is that the tag coincides with the conda package version (and hasn't changed in a year), so I was thinking of doing an exception for this one, what do you think?

Reply:
Works for me! :+1: